components/AuthenticationGate/AuthenticationGate.tsx

```diff
@@ -7,7 +7,7 @@ function loginWithDiscord() {
   return signIn("discord");
 }
 
-const AuthenticationGate = ({ sessionStatus, children }) => {
+const AuthenticationGate: React.FunctionComponent = ({ sessionStatus, children }: any) => {
```

Comment:
Instead of `any` an actual type for props could be used here:

```suggestion
type Props = {
  sessionStatus: string
}

const AuthenticationGate: React.FunctionComponent<Props> = ({ sessionStatus, children }) => {
```